communicator/ssh/connect_test.go

```diff
@@ -0,0 +1,65 @@
+package ssh_test
+
+import (
+	"bytes"
+	"fmt"
+	helperssh "github.com/hashicorp/packer/helper/ssh"
+	"golang.org/x/crypto/ssh"
+	"log"
+	"os/user"
+	"testing"
+	"time"
+)
+
+func getIdentityCertFile() (certSigner ssh.Signer, err error) {
+	usr, _ := user.Current()
+	privateKeyFile := usr.HomeDir + "/.ssh/id_ed25519"
+	certificateFile := usr.HomeDir + "/.ssh/id_ed25519-cert.pub"
+
+	return helperssh.FileSignerWithCert(privateKeyFile, certificateFile)
+}
+
+// Rename to Uppercase to test and change host constant
+func testBastionConnectFunc(t *testing.T) {
```

Comment:
Not sure -- how to test this without having things setup properly in an SSH environment.  The lint checks are failing.  Should I just remove this file?

Reply:
Keep the file for now; but we can put it behind an acceptance test flag: 
```
	if os.Getenv("PACKER_ACC") == "" {
		t.Skip("This test is only run with PACKER_ACC=1")
	}
```

Reply:
Fixed!  Thanks for the pointer.